iscsi/targetd/ansible/provisioner-playbook.yaml

```diff
@@ -67,6 +67,21 @@
        register: iscsi_role
        when: system_role.rc == 1
 
+     - name: Add system:persistent-volume-provisioner role to service account
```

Comment:
this change looks wrong; "Add system:persistent-volume-provisioner role to service account" already happens above. rebasing error?

Reply:
yes my bad, something went wrong during the rebase.